packages/config-loader/api-report.md

```diff
@@ -21,6 +21,8 @@ export type AsyncConfigSourceGenerator = AsyncGenerator<
 
 // @public
 export interface BaseConfigSourcesOptions {
+  // (undocumented)
+  disableConfigWatching?: boolean;
```

Comment:
Could you make all of these `watch?: boolean` instead? and then have it default to `true`. It's a bit backwards to default to `true`, but that way if we want to add watch options in the future we can make it `watch?: boolean | WatchOptions`